development/Dockerfile

```diff
@@ -1,15 +1,16 @@
-ARG NAUTOBOT_VER="1.0.1"
+ARG NAUTOBOT_VER="1.3.5"
 ARG PYTHON_VER=3.8
```

Comment:
I am wondering if there is a way to tokenize these so we can have them in one place?

Reply:
If it is, I would like that to be a separate PR. This is just to drop 3.6, not to improve CI.

Reply:
Agreed.